### DISCUSSION TITLE
How to get more than 100 tracks in a playlist

### DISCUSSION THREAD

**Discussion by KenthJ** [JohnnyCrazy/SpotifyAPI-NET#621] [Q&A]
spotify.Playlists.Get("playlistId"); only returns the first 100 tracks

How to get the next ones?

In an old thread a pagination helper was mentioned, but the link in there is dead, and I can't find any other info about this.

Any help is greatly appreciated.

**Comment by FrankBJensen** [comment]
Hi Kenth.

As far as I know, you have to load 100 at a time, like this. Increment offset by 100 until no tracks are returned

`Paging<PlaylistTrack<IPlayableItem>> pUserTracks = await spotify.Playlists.GetItems(id, new PlaylistGetItemsRequest { Offset = 0 });`

Frank

**Reply by KenthJ** [reply]
Thanks a lot, that solved the problem
